app/main.py

```
@@ -1 +1,32 @@
-# write your code here
+from .cafe import Cafe
+from .errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    not_vaccinated_count = 0
+    not_wearing_mask_count = 0
+
+    has_vaccine_issue = False
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+
+        except VaccineError:
+            not_vaccinated_count += 1
+            has_vaccine_issue = True
+
+        except NotWearingMaskError:
+            not_wearing_mask_count += 1
+
+        except Exception:
```

Catching the broad `Exception` (`except Exception:`) will hide unexpected errors and can change the function behavior by treating any unexpected error as a vaccine issue. The task requires using `VaccineError` to catch vaccination-related problems; prefer removing this broad `except` or handling only the expected exceptions explicitly so other errors surface during testing/debugging.